coalib/settings/ConfigurationGathering.py

```diff
@@ -61,11 +61,12 @@ def load_config_file(filename, log_printer, silent=False):
                 log_printer.warn('The default coafile {0!r} was not found. '
                                  'You can generate a configuration file with '
                                  'your current options by adding the `--save` '
-                                 'flag.'.format(Constants.default_coafile))
+                                 'flag or suppress any use of config files with `-I`.'.format(Constants.default_coafile))
```

Comment:
Also look at the comment made by gitmate-bot.
Read all the links. Its important for you to learn about all that. :)

The change otherwise seems fine.